### ISSUE TITLE
Microsoft Login causes white screen (of death)

### ISSUE THREAD

**Issue by Trikzon** [Ellpeck/ObsidianCustomFrames#11]
When attempting to make a custom frame for Canvas, my school's online class portal, it attempts to login with Microsoft and results in a white screen. My school's site is `https://usflearn.instructure.com`, but the same error can be seen by going to `https://www.outlook.com` and pressing 'Sign in'.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/29845000/160692106-c4d8c4a4-65f0-4789-9c96-65990cc39cc3.png">

```
Unsafe attempt to initiate navigation for frame with origin 'app://obsidian.md' from frame with URL 'https://login.live.com/login.srf?wa=wsignin1.0&rpsnv=13&ct=1648582097&rver=7.0.6737.0&wp=MBI_SSL&wreply=https%3a%2f%2foutlook.live.com%2fowa%2f%3fnlp%3d1%26RpsCsrfState%3d85b474ae-4021-9bbb-74c5-a264c2f1ef84&id=292841&aadredir=1&CBCXT=out&lw=1&fl=dob%2cflname%2cwld&cobrandid=90015'. The frame attempting navigation of the top-level window is sandboxed, but the flag of 'allow-top-navigation' or 'allow-top-navigation-by-user-activation' is not set.

Unsafe attempt to initiate navigation for frame with origin 'app://obsidian.md' from frame with URL 'https://login.live.com/login.srf?wa=wsignin1.0&rpsnv=13&ct=1648582097&rver=7.0.6737.0&wp=MBI_SSL&wreply=https%3a%2f%2foutlook.live.com%2fowa%2f%3fnlp%3d1%26RpsCsrfState%3d85b474ae-4021-9bbb-74c5-a264c2f1ef84&id=292841&aadredir=1&CBCXT=out&lw=1&fl=dob%2cflname%2cwld&cobrandid=90015'. The frame attempting navigation of the top-level window is sandboxed, but the flag of 'allow-top-navigation' or 'allow-top-navigation-by-user-activation' is not set.

DevTools failed to load source map: Could not load content for https://logincdn.msauth.net/shared/1.0/content/js/oneDs_cf88713273157e0b2931.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

**Comment by Ellpeck** [comment]
What version of the plugin are you on? This issue should have been fixed in 2.2.0.

**Comment by Trikzon** [comment]
... and I just realized that this plugin says it requires 0.14.3, not 0.14.2. I'm on 2.1.0. Guess I have to wait longer. Thanks for making this, am excited to have everything in Obsidian
